variables.tf

```diff
@@ -100,13 +100,44 @@ variable "resource_names_map" {
 variable "vpc_id" {
   description = "The VPC ID of the VPC where infrastructure will be provisioned"
   type        = string
+  default     = null
+}
+
+variable "vpc_name" {
```

Comment:
Remove the default, make it null. and update the description that this is required when create_vpc=true